src/Toml/Printer.hs

```diff
@@ -26,17 +26,19 @@ import qualified Data.Text as Text
 
 {- | Configures the pretty printer. -}
 data PrintOptions = PrintOptions
-    { shouldSort :: Bool  -- ^ should table keys be sorted or not
-    , indent     :: Int   -- ^ indentation size
-    } deriving (Show)
+    {- | How table keys should be sorted, if at all. -}
+    { printOptionsSorting :: !(Maybe (Key -> Key -> Ordering))
+    {- | Number of spaces by which to indent. -}
+    , printOptionsIndent  :: !Int
```

Comment:
Oh, I just noticed. Documentation here is formatted a little bit wrong. It should be:

```haskell
{- | Configures the pretty printer. -}
data PrintOptions = PrintOptions
    { -- | How table keys should be sorted, if at all. 
    , printOptionsSorting :: !(Maybe (Key -> Key -> Ordering))

      -- | Number of spaces by which to indent.
    , printOptionsIndent  :: !Int
    }
```